lib/flashcard_runner1.rb

```diff
@@ -0,0 +1,32 @@
+require_relative 'card'
+require_relative 'deck'
+require_relative 'turn'
+require_relative 'round'
+require_relative 'card_generator'
+
+filename = "cards.txt"
+cards = CardGenerator.new(filename).cards
+deck = Deck.new(cards)
+round = Round.new(deck)
+
+def start(round)
+  count = 1
+  puts "Welcome! You're playing with #{round.deck.count} cards."
```

Comment:
Great use of interpolation here, that makes your game dynamic. No matter how many cards are in the deck, the correct number always prints.